src/app/ApplicationTemplate.Business/DadJokes/DadJokesService.cs

```diff
@@ -70,10 +70,10 @@ public async Task SetIsFavorite(CancellationToken ct, DadJokesQuote quote, bool
 
 			var settings = await _applicationSettingsService.GetCurrent(ct);
 
-			quote = quote.WithIsFavorite(isFavorite);
+			quote = quote with { IsFavorite = isFavorite };
```

Comment:
Space here ? Is it a new code style ?

Reply:
Yes, [with expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/with-expression)